Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -24,6 +24,10 @@
     <string name="wiimote_extension_3">Wii Remote Extension 4</string>
 
     <string name="real_balance_board">Real Balance Board</string>
+    <string name="dualshockudp_client">DualshockUDP Client (Background Input)</string>
```

Comment:
We call it DSU in all user-facing strings, not DualshockUDP. Also, I don't see what the Background Input setting has to do with it.